hazelcast/src/main/java/com/hazelcast/instance/impl/NodeContext.java

```diff
@@ -37,5 +38,5 @@
     Joiner createJoiner(Node node);
 
     // TODO Consider the changes here (JET?)
```

Comment:
@vbekiaris Do you remember why there is this `TODO` comment?

Reply:
Seems stale, it was left there since the advanced network implementation PR to check whether Hazelcast Jet (that resided in a separate repository at the time) was affected.

Reply:
removed it

Reply:
Thanks, Ufuk.